includes/wc-core-functions.php

```diff
@@ -816,8 +815,7 @@ function get_woocommerce_currency_symbols() {
 			'USD' => '&#36;',
 			'UYU' => '&#36;',
 			'UZS' => 'UZS',
-			'VEF' => 'Bs F',
-			'VES' => 'Bs.S',
+			'VES' => 'Bs.',
```

Comment:
Same here. Would it make sense to keep VEF as it is and change VES to 'Bs.'?

Reply:
Yes, I agree with your suggestion here too 👍